### PR TITLE
changes in db layer

### PR DESCRIPTION
**Pull request by yinonbz**
changes in db layer and db handler

### CODE REVIEW

File: src/main/java/dataLayer/DBHandler.java [hunk 1]
```diff
@@ -44,7 +98,7 @@ public Object selectFromDB(String objectName, String table) {
      * @return
      */
     @Override
-    public boolean removeFromDB(String objectName, String table) {
+    public boolean removeFromDB(String objectName, String tableת,String where,String groupby) {
```
Comment (tomersein) [inline src/main/java/dataLayer/DBHandler.java]: hebew character

File: src/main/java/dataLayer/DBHandler.java [hunk 2]
```diff
@@ -17,9 +29,51 @@ public DBHandler(){
      */
 
     @Override
-    public boolean containInDB(String objectName,String table) {
+    public boolean containInDB(String objectName,String table,String where,String groupby) {
         //create sql query to search record in db using ObjectName
-        //return the result
+        String query = "Select" +objectName +"from" +table;
```
Comment (tomersein) [inline src/main/java/dataLayer/DBHandler.java:34]: very nice

**Review by tomersein — CHANGES_REQUESTED** [review]
please fix the bug

File: src/main/java/dataLayer/DBHandler.java (hunk 1 above)
Comment (yinonbz) [reply src/main/java/dataLayer/DBHandler.java]: fixed

**Review by tomersein — APPROVED** [review]
connection to DB is approved by Tomer

**Pull request merged**